aws/storagemanager/aws.go

```diff
@@ -82,7 +82,7 @@ func (a *awsStorageManager) RecommendStoragePoolUpdate(
 func determineIOPSForPool(instStorage *cloudops.StoragePoolSpec, row *cloudops.StorageDecisionMatrixRow, currentIOPS uint64) uint64 {
 	if instStorage.DriveType == DriveTypeGp2 {
 		return instStorage.DriveCapacityGiB * Gp2IopsMultiplier
-	} else if instStorage.DriveType == DriveTypeIo1 {
+	} else if instStorage.DriveType == DriveTypeIo1 || instStorage.DriveType == DriveTypeGp3 {
```

Comment:
how about io2 types?

If it's low hanging, let's add a UT. Okay to skip here as this is a trivial function.

Reply:
We don't support io2 today.
```
  drive_type: gp2
  drive_type: gp3
  drive_type: io1

```
We have unit tests in aws_test.go. It seem to have caught one scenario. I am testing that on an actual cluster.

Reply:
Added a unit test. The failure was a test parameter failure not a product issue.